src/Microsoft.DotNet.Build.Tasks/PackageFiles/ApiCompat.runtimeconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "runtimeOptions": {
+    "framework": {
+      "name": "Microsoft.NETCore.App",
+      "version": "1.0.0-rc3-002733"
```

Comment:
Sounds like we are fine with keeping this version as-is then. We can upgrade it to 1.0.0 stable when available.